### PR TITLE
Networking updates

### PR DESCRIPTION
**Pull request by adamgoth**
Remove function that syncs maker object with account

Remove network from query params (just rely on active provider connection instead)

Add default provider wrapper when there is no connector injected so that `useActiveWeb3React` still works

Add connect chain name to header

### CODE REVIEW

File: modules/app/components/layout/header/NetworkAlertModal.tsx [hunk 1]
```diff
@@ -73,6 +81,8 @@ const NetworkAlertModal = ({
               Your wallet is connected to an unsupported network, please switch it to{' '}
               {SupportedNetworks.MAINNET} or {SupportedNetworks.GOERLI} to continue.
             </Text>
+            {/* <Button onClick={() => handleSwitchNetwork(SupportedNetworks.MAINNET)}>Switch to mainnet</Button>
```
Comment (b-pmcg) [inline modules/app/components/layout/header/NetworkAlertModal.tsx:84]: nice, kinda happy to get rid of this. Also, just flagging the `network mismatch` error above, probably need to adjust the code there too.

**Review by b-pmcg — APPROVED** [review]
looks good, would it make sense to call the `chainIdToNetworkName` function inside the `useActiveWeb3React` hook and return the network name there?

**Comment by adamgoth** [comment]
> looks good, would it make sense to call the `chainIdToNetworkName` function inside the `useActiveWeb3React` hook and return the network name there?

Had the same thought. I think it would make it cleaner to use for sure. I only hesitated since right now we just return the context unaltered from web3-react. But I don’t think it’s an issue to add in network too and return it.

**Comment by b-pmcg** [comment]
also I love the network indicator, we should just leave it like that and not tell @tiago-varandas  xD

**Pull request merged**